modified_files/net/net.gypi

```diff
@@ -817,6 +817,8 @@
       'http2/hpack/decoder/hpack_block_decoder.h',
       'http2/hpack/decoder/hpack_decoder_string_buffer.cc',
       'http2/hpack/decoder/hpack_decoder_string_buffer.h',
+#      'http2/hpack/decoder/hpack_decoder_tables.cc',
```

Comment:
HPACK files are in http2 directory, their tests might be also in http2.
I didn't remove hpack_decoder_tables* file names from net.gypi, they are still there commented. But I removed these files from this pull request because QUIC doesn't rely on these files to build.